src/components/AboutUsIcons/AboutUsIcons.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Image from 'next/image';
+
+type AboutUsIconProps = {
+  src: string;
+  alt?: string;
+  text: string;
+};
+
+export const AboutUsIcon: React.FC<AboutUsIconProps> = ({ src, alt = 'About Us', text }) => {
```

Comment:
Please, don't use `FC` from React. It became outdates recently. Set the type like this:

`export const AboutUsIcon = ({ src, alt = 'About Us', text }: AboutUsIconProps) => {`